src/components/Calendar/CalendarPage.scss

```diff
@@ -0,0 +1,98 @@
+.calendar {
+  border: 1px solid lightgray;
+
+  &__item {
+    border: 1px solid lightgray;
```

Comment:
You can replace this with
.calendar {
border: 2px solid light gray; 
}
The result is the same